.github/workflows/scripts/parse_changelog.rb

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/ruby
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'date'
+require_relative '../../../lib/new_relic/version'
+
+DIVIDER = '---'
+SUPPRORT_STATEMENT = "<Callout variant=\"important\">
```

Comment:
`SUPPORT` is misspelled here.

Reply:
[fixed](https://github.com/newrelic/newrelic-ruby-agent/pull/1897/commits/7bc24cab4c3846e65e89b842d2f9d2195e0c3629#diff-244d6a89554be30d0a4af9f69702776fe95737ee944a074750ea2fcd22657c0eR11)